include/revng-c/mlir/Dialect/Clift/Transforms/Passes.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng-c/mlir/Dialect/Clift/Transforms/ModelOption.h"
+
+namespace mlir::clift {
+
+#define GEN_PASS_DECL
+#include "revng-c/mlir/Dialect/Clift/Transforms/Passes.h.inc"
+
+std::unique_ptr<OperationPass<mlir::ModuleOp>> createImportModelTypesPass();
+
+#define GEN_PASS_REGISTRATION
+#include "revng-c/mlir/Dialect/Clift/Transforms/Passes.h.inc"
```

Comment:
So this is necessary to define an `OperationPass` and register it. And it's including an `.inc` file that is generated by tablegen by de declarative stuff in `include/revng-c/mlir/Dialect/Clift/Transforms/Passes.td`. Is that right?

I'm taking this as an opportunity to understand all the mlir tricks and make sure I remember them.

Reply:
Right. It's just including the tablegen generated stuff.